cards.py

```diff
@@ -1,17 +1,13 @@
 # get_card()
 import random
 
-deck = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K", "A"]
-
 def get_card():
     """
     Parameter:
         none
     Return:
         <String> a random value from a list of a deck of cards' ranks
     """
-    # print(random.choice(deck))
+    deck = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "J", "Q", "K", "A"]
```

Comment:
I have to reread this problem, but going off of memory, should we just have the deck be an int and string list? Meaning 1-10 is an int while J-A are strings. That way we can just go ahead and compare ints from the get-go and only have to worry about J-A